docs/vm/create-windows-vm.md

```diff
@@ -176,12 +179,10 @@ Alternately if the system has already booted into the UEFI shell, you can type i
 
 There is a known issue with Windows VM when it is allocated more than 8GiB without enough reserve memory configured. The VM crashes without warning.
 
-This can be fixed by allocating at least 256MiB of reserved memory to the template on the Advanced Options tab.
+This can be fixed by allocating at least 256MiB of reserved memory to the template on the Advanced Options tab. If 256MiB doesn't work, try 512MiB.
```

Comment:
```suggestion
This can be fixed by allocating at least 256MiB of reserved memory to the template on the Advanced Options tab. If `256MiB` doesn't work, try `512MiB`.
```